src/code/z_parameter.c

```diff
@@ -132,36 +1222,170 @@
+};
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_80119610.s")
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_8011B4E0.s")
 
+u16 D_801BFC50[] = {
+    0xC000, 0xE000, 0x0000, 0x2000, 0xA000, 0x8000, 0x6000, 0x4000,
+};
+s16 D_801BFC60[][3] = {
+    255, 255, 255, 255, 165, 55,
+};
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_8011B5C0.s")
 
+s16 D_801BFC6C[] = {
+    0x004E, 0x0036, 0x001D, 0x0005, 0xFFEE, 0xFFD6, 0xFFBD, 0xFFAB,
+};
+s16 D_801BFC7C[] = {
+    0x00B4, 0x00B4, 0x00B4, 0x00B4, 0xFF4C, 0xFF4C, 0xFF4C, 0xFF4C,
+};
+s16 D_801BFC8C[2][3] = {
+    { 0xFF, 0xFF, 0xFF },
+    { 0xFF, 0xA5, 0x37 },
+};
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_8011B9E0.s")
 
+s16 D_801BFC98[] = {
+    0x004E, 0x0036, 0x001D, 0x0005, 0xFFEE, 0xFFD6, 0xFFBD, 0xFFAB,
+};
+u16 D_801BFCA8[] = {
+    0xC000, 0xE000, 0x0000, 0x2000, 0xA000, 0x8000, 0x6000, 0x4000,
+};
+s16 D_801BFCB8[2][3] = {
+    { 0xFF, 0xFF, 0xFF },
+    { 0xFF, 0xA5, 0x37 },
+};
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_8011BF70.s")
 
+TexturePtr D_801BFCC4[] = {
+    gMinigameLetterPTex, gMinigameLetterETex, gMinigameLetterRTex, gMinigameLetterFTex,
+    gMinigameLetterETex, gMinigameLetterCTex, gMinigameLetterTTex, gMinigameExclamationTex,
+};
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_8011C4C4.s")
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_8011C808.s")
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_8011C898.s")
 
+s16 D_801BFCE4 = 0;
+s16 D_801BFCE8[] = { 0, 0, 0, 0, 0, 0, 0, 0 };
+s16 D_801BFCF8 = 0x63;
+s16 D_801BFCFC[] = {
+    // timer digit width
+    0x10, 0x19, 0x22, 0x2A, 0x33, 0x3C, 0x44, 0x4D,
+};
+s16 D_801BFD0C[] = {
+    // digit width
+    9, 9, 8, 9, 9, 8, 9, 9,
+};
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_8011CA64.s")
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_8011E3B4.s")
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_parameter/func_8011E730.s")
 
+// rupeeDigitsFirst
+s16 D_801BFD1C[] = { 1, 0, 0, 0 };
+// rupeeDigitsCount
+s16 D_801BFD24[] = { 2, 3, 3, 0 };
+// rupeeIconPrimColor
+Color_RGB16 D_801BFD2C[] = {
+    { 200, 255, 100 },
+    { 170, 170, 255 },
+    { 255, 105, 105 },
+};
+// rupeeIconEnvColor
+Color_RGB16 D_801BFD40[] = {
+    { 0, 80, 0 },
+    { 10, 10, 80 },
+    { 40, 10, 0 },
+};
+// minigameCountdownTexs
+TexturePtr D_801BFD54[] = {
+    gMinigameCountdown3Tex,
+    gMinigameCountdown2Tex,
+    gMinigameCountdown1Tex,
+    gMinigameCountdownGoTex,
+};
+// minigameCountdownTexHeights
+s16 D_801BFD64[] = { 24, 24, 24, 40 };
+// minigameCountdownPrimColor
+Color_RGB16 D_801BFD6C[] = {
+    { 100, 255, 100 },
+    { 255, 255, 60 },
+    { 255, 100, 0 },
+    { 120, 170, 255 },
+};
+// grandma's story pictures
+s32 D_801BFD84[] = {
+    0x07000000,
+    0x07012C00,
+};
+// grandma's story TLUT
+u32 D_801BFD8C[] = {
+    0x07025800,
+    0x07025A00,
+};
```

Comment:
These are already in the repo, so you can add them now.

Reply:
I think they need to be put into spec for them to be included, I'll update this once the do-static PR is in

Reply:
Updated to include these now